src/pages/index.fr-fr.js

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import Layout from "../layouts";
+import { graphql, Link } from 'gatsby'
+import { BlogLatest, BlogRoll, IndexHelmet, MainTitle, Section, Videos } from "../common/Index";
+import "../stylesheets/index.scss";
+
+const tracingText = `
+La faculté d'attacher des programmes eBPF sur des « trace points » et des
+« probe points », que ce soit dans le noyau ou dans les applications
+utilisateurs, offre des possibilités inégalées en termes de visibilité. Les
```

Comment:
I'm really unsure for my proposed change, but I wanted to remove `faculté` as I think it has a slightly different meaning than intended here.
```suggestion
La possibilité d'attacher des programmes eBPF sur des « trace points » et des
« probe points », que ce soit dans le noyau ou dans les applications
utilisateurs, ouvre des opportunités inégalées en termes de visibilité. Les
```

Reply:
Changed to
```
La possibilité d'attacher des programmes eBPF sur des « trace points » et des
« probe points », que ce soit dans le noyau ou dans les applications
utilisateurs, apporte une dimension nouvelle en termes de visibilité.
```